### PR TITLE
Update code samples based on SDK API changes.

### PR DESCRIPTION
**Pull request by FeiChen-okta**
## Description:
Code samples are out of date since the latest okta-oidc-android changes.
This PR updates the code samples in the developer docs to be in sync with
https://github.com/okta/okta-oidc-android/

Add custom-sign-in guide.

### Resolves:

* [OKTA-222098](https://oktainc.atlassian.net/browse/OKTA-222098)

### CODE REVIEW

File: packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-custom-ui/android/signin.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+In this example we will show how to create a custom UI for user sign-in. First we will create a simple dialog to ask for username and password:
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-custom-ui/android/signin.md]: @FeiChen-okta Sorry, meant to sync with you on this: can you remove this from `sign-into-mobile-app` and place it in this guide instead? https://github.com/okta/okta-developer-docs/pull/333

File: packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-custom-ui/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-custom-ui/index.md]: Move to https://github.com/okta/okta-developer-docs/pull/333

File: packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-out-page/android/signout.md [hunk 3]
```diff
@@ -6,10 +6,11 @@ If the user is signed in using the browser-initiated authorization flow, then lo
 
 ### Clear Browser Session
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-out-page/android/signout.md]: Can you move this to the Sign Out Guide?: https://developer.okta.com/guides/sign-users-out/aspnet/before-you-begin/

**Comment by nbarbettini** [comment]
Code updates look good ✔️ 

@FeiChen-okta  Before we merge this, we need to move
* Custom Sign-in UI guide to its own branch (PR to Ildar's WIP)
* Sign Out content to its own branch (adding iOS to the existing Sign Out guide)

Then the code example changes can be merged to `nb-mobile-guide-phase2`.

**Comment by FeiChen-okta** [comment]
OK will make the changes.

**Comment by nbarbettini** [comment]
Thanks! Merging

**Pull request merged**